### PR TITLE
Added node http server

### PR DESCRIPTION
**Pull request by radu1122**
# Pull Request Template

I added the functionality to import an HTTP server (express, fastify, etc) without any changes. The only modification is in the yaml with `type: httpServer`

## Type of change

-   [x] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [ ] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Checklist

-   [ ] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [ ] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Review by stefandarius — APPROVED** [review]

File: src/projectConfiguration/yaml/v2.ts [hunk 1]
```diff
@@ -93,7 +93,8 @@ function parseGenezioConfig(config: unknown) {
             return nameRegex.test(value);
         }, "Must start with a letter and contain only letters, numbers and dashes."),
         path: zod.string(),
-        handler: zod.string(),
+        // handler is mandatory only if type is AWS
+        handler: zod.string().default("handler"),
```
Comment (andreia-oca) [inline src/projectConfiguration/yaml/v2.ts:97]: This should be `.optional()` and checked with a `.refine()` condition.

**Review by andreia-oca — APPROVED** [review]
Looks good! 🎉

File: src/functionHandlerProvider/providers/HttpServerHandlerProvider.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    myHandler(req, res);
+  }).catch((error) => {
+    console.error(error);
+    throw error;
+  });
+}
+
+const handler = async function(event) {
+
+  const timeStartBefore = new Date();
```
Comment (vladiulianbogdan) [inline src/functionHandlerProvider/providers/HttpServerHandlerProvider.ts:107]: These two variables are not used.

File: src/functionHandlerProvider/providers/HttpServerHandlerProvider.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    const timeEndAfter = new Date();
+
+
+    // Return the appropriate response in the AWS Lambda format
+    return {
+      statusCode: response.statusCode,
+      headers: response.headers,
+      body: response.body,
+    };
+  } catch (error) {
```
Comment (vladiulianbogdan) [inline src/functionHandlerProvider/providers/HttpServerHandlerProvider.ts:121]: You should let the runtime handle the errors

**Review by vladiulianbogdan — APPROVED** [review]

**Pull request merged**